src/main/java/com/currencyfair/onesignal/model/notification/Notification.java

```diff
@@ -1216,6 +1219,12 @@ public Boolean getChrome() {
         return chrome;
     }
 
+    public void setExternalId(String string) {
+        this.externalId = string;
+    }
+
+    public String getExternalId() { return this.externalId; }
```

Comment:
nit, this should be split into 3 lines, to match the formatting of the other methods in this file.
```java
public String getExternalId() {
    return this.externalId;
}
```

Reply:
Agree with this, that is why build is failing.

Reply:
Formatting looks good now